### ISSUE TITLE
Complete windows install instructions

### ISSUE THREAD

**Issue by mkassner** [pupil-labs/cygl#3]
@MichaelBarz we still need instructions here. Can you take care of that?

**Comment by fneitzel** [comment]
In the moment i try to handle this issue:

C:\Users\testing\git\pyglui\pyglui\cygl2>python setup.py install
'glew.pxd' is up-to-date.
C:\Python27\lib\distutils\dist.py:251: UserWarning: 'licence' distribution option is deprecated; use 'license'
  warnings.warn(msg)
C:\Python27\lib\distutils\dist.py:267: UserWarning: Unknown distribution option: 'exclude_package_data'
  warnings.warn(msg)
running install
running build
running build_py
running build_ext
building 'cygl.utils' extension
creating build\temp.win32-2.7
creating build\temp.win32-2.7\Release
C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\BIN\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG -Iwin_glew -IC:\Python27\include -IC:\Python27\PC /Tcutil
s.c /Fobuild\temp.win32-2.7\Release\utils.obj
utils.c
C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\BIN\link.exe /DLL /nologo /INCREMENTAL:NO /LIBPATH:win_glew /LIBPATH:C:\Python27\libs /LIBPATH:C:\Python2
7\PCbuild glew32.lib openGL32.lib /EXPORT:initutils build\temp.win32-2.7\Release\utils.obj /OUT:build\lib.win32-2.7\cygl\utils.pyd /IMPLIB:build\temp.win32-2.7\
Release\utils.lib /MANIFESTFILE:build\temp.win32-2.7\Release\utils.pyd.manifest
   Bibliothek "build\temp.win32-2.7\Release\utils.lib" und Objekt "build\temp.win32-2.7\Release\utils.exp" werden erstellt.
utils.obj : error LNK2019: Verweis auf nicht aufgelöstes externes Symbol "__imp__glewInit@0" in Funktion "___pyx_f_6pyglui_5cygl2_5utils_init".
build\lib.win32-2.7\cygl\utils.pyd : fatal error LNK1120: 1 nicht aufgelöste externe Verweise.
error: command 'C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\BIN\link.exe' failed with exit status 1120

Andybody help please?

**Comment by fneitzel** [comment]
Hi, again.

I installed on another System without Visual Studio 10. (VS9 installed) and this error occurs:

C:\Users\pyglui\pyglui\cygl>python setup.py install
generating glew.pxd based on 'win_glew/gl/glew.h'
Compiling utils.pyx because it changed.
Compiling shader.pyx because it changed.
Cythonizing shader.pyx
Cythonizing utils.pyx
C:\Python27\lib\distutils\dist.py:251: UserWarning: 'licence' distribution optio
n is deprecated; use 'license'
  warnings.warn(msg)
C:\Python27\lib\distutils\dist.py:267: UserWarning: Unknown distribution option:
 'exclude_package_data'
  warnings.warn(msg)
running install
running build
running build_py
creating build
creating build\lib.win-amd64-2.7
creating build\lib.win-amd64-2.7\cygl
copying glew_pxd.py -> build\lib.win-amd64-2.7\cygl
copying **init**.py -> build\lib.win-amd64-2.7\cygl
running build_ext
building 'cygl.utils' extension
error: Unable to find vcvarsall.bat

What's the correct c++ compiler to use?

I already tried:

http://stackoverflow.com/questions/2817869/error-unable-to-find-vcvarsall-bat

```
Visual Studio 2010 (VS10): SET VS90COMNTOOLS=%VS100COMNTOOLS%
Visual Studio 2012 (VS11): SET VS90COMNTOOLS=%VS110COMNTOOLS%
Visual Studio 2013 (VS12): SET VS90COMNTOOLS=%VS120COMNTOOLS%
```

**Comment by mkassner** [comment]
@MichaelBarz do you think you can make a wheel for pyglui?

**Comment by MichaelBarz** [comment]
I think I can manage that until the end of the week.

**Comment by fneitzel** [comment]
@MichaelBarz That would be great!

Currently i found a hint for a work-around: http://stackoverflow.com/questions/22213558/unresolved-external-symbol-imp-glewinit-vs-imp-glewinit0

I downloaded the latest glew (1.12.0) - So I finally could compile cygl on windows.

Another issue was about VS9. It has no 'stdin.h' included as required for compiling in 'utils.c' 
I downloaded it here: https://code.google.com/p/msinttypes/downloads/list
